Bottom: fc1641c4347c1b027ee862d5eff3749f8c010dcd
Top:    0df9bfaeac1657018b37f26031005e769e2fc343
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 15:57:04 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/Vagrantfile b/Vagrantfile
index af48105..3e88653 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -25,7 +25,5 @@ Vagrant.configure("2") do |config|
         worker.vm.provider "virtualbox" do |vb|
             vb.name = "bbworker"
         end
-
     end
-
 end
